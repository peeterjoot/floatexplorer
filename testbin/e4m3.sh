./floatexplorer --e4m3 0x00
./floatexplorer --e4m3 0x01
./floatexplorer --e4m3 0x02
./floatexplorer --e4m3 0x03
./floatexplorer --e4m3 0x04
./floatexplorer --e4m3 0x05
./floatexplorer --e4m3 0x06
./floatexplorer --e4m3 0x07
./floatexplorer --e4m3 0x08
./floatexplorer --e4m3 0x09
./floatexplorer --e4m3 0x0A
./floatexplorer --e4m3 0x0B
./floatexplorer --e4m3 0x0C
./floatexplorer --e4m3 0x0D
./floatexplorer --e4m3 0x0E
./floatexplorer --e4m3 0x0F
./floatexplorer --e4m3 0x10
./floatexplorer --e4m3 0x11
./floatexplorer --e4m3 0x12
./floatexplorer --e4m3 0x13
./floatexplorer --e4m3 0x14
./floatexplorer --e4m3 0x15
./floatexplorer --e4m3 0x16
./floatexplorer --e4m3 0x17
./floatexplorer --e4m3 0x18
./floatexplorer --e4m3 0x19
./floatexplorer --e4m3 0x1A
./floatexplorer --e4m3 0x1B
./floatexplorer --e4m3 0x1C
./floatexplorer --e4m3 0x1D
./floatexplorer --e4m3 0x1E
./floatexplorer --e4m3 0x1F
./floatexplorer --e4m3 0x20
./floatexplorer --e4m3 0x21
./floatexplorer --e4m3 0x22
./floatexplorer --e4m3 0x23
./floatexplorer --e4m3 0x24
./floatexplorer --e4m3 0x25
./floatexplorer --e4m3 0x26
./floatexplorer --e4m3 0x27
./floatexplorer --e4m3 0x28
./floatexplorer --e4m3 0x29
./floatexplorer --e4m3 0x2A
./floatexplorer --e4m3 0x2B
./floatexplorer --e4m3 0x2C
./floatexplorer --e4m3 0x2D
./floatexplorer --e4m3 0x2E
./floatexplorer --e4m3 0x2F
./floatexplorer --e4m3 0x30
./floatexplorer --e4m3 0x31
./floatexplorer --e4m3 0x32
./floatexplorer --e4m3 0x33
./floatexplorer --e4m3 0x34
./floatexplorer --e4m3 0x35
./floatexplorer --e4m3 0x36
./floatexplorer --e4m3 0x37
./floatexplorer --e4m3 0x38
./floatexplorer --e4m3 0x39
./floatexplorer --e4m3 0x3A
./floatexplorer --e4m3 0x3B
./floatexplorer --e4m3 0x3C
./floatexplorer --e4m3 0x3D
./floatexplorer --e4m3 0x3E
./floatexplorer --e4m3 0x3F
./floatexplorer --e4m3 0x40
./floatexplorer --e4m3 0x41
./floatexplorer --e4m3 0x42
./floatexplorer --e4m3 0x43
./floatexplorer --e4m3 0x44
./floatexplorer --e4m3 0x45
./floatexplorer --e4m3 0x46
./floatexplorer --e4m3 0x47
./floatexplorer --e4m3 0x48
./floatexplorer --e4m3 0x49
./floatexplorer --e4m3 0x4A
./floatexplorer --e4m3 0x4B
./floatexplorer --e4m3 0x4C
./floatexplorer --e4m3 0x4D
./floatexplorer --e4m3 0x4E
./floatexplorer --e4m3 0x4F
./floatexplorer --e4m3 0x50
./floatexplorer --e4m3 0x51
./floatexplorer --e4m3 0x52
./floatexplorer --e4m3 0x53
./floatexplorer --e4m3 0x54
./floatexplorer --e4m3 0x55
./floatexplorer --e4m3 0x56
./floatexplorer --e4m3 0x57
./floatexplorer --e4m3 0x58
./floatexplorer --e4m3 0x59
./floatexplorer --e4m3 0x5A
./floatexplorer --e4m3 0x5B
./floatexplorer --e4m3 0x5C
./floatexplorer --e4m3 0x5D
./floatexplorer --e4m3 0x5E
./floatexplorer --e4m3 0x5F
./floatexplorer --e4m3 0x60
./floatexplorer --e4m3 0x61
./floatexplorer --e4m3 0x62
./floatexplorer --e4m3 0x63
./floatexplorer --e4m3 0x64
./floatexplorer --e4m3 0x65
./floatexplorer --e4m3 0x66
./floatexplorer --e4m3 0x67
./floatexplorer --e4m3 0x68
./floatexplorer --e4m3 0x69
./floatexplorer --e4m3 0x6A
./floatexplorer --e4m3 0x6B
./floatexplorer --e4m3 0x6C
./floatexplorer --e4m3 0x6D
./floatexplorer --e4m3 0x6E
./floatexplorer --e4m3 0x6F
./floatexplorer --e4m3 0x70
./floatexplorer --e4m3 0x71
./floatexplorer --e4m3 0x72
./floatexplorer --e4m3 0x73
./floatexplorer --e4m3 0x74
./floatexplorer --e4m3 0x75
./floatexplorer --e4m3 0x76
./floatexplorer --e4m3 0x77
#./floatexplorer --e4m3 0x78
./floatexplorer --e4m3 0x79
./floatexplorer --e4m3 0x7A
./floatexplorer --e4m3 0x7B
./floatexplorer --e4m3 0x7C
./floatexplorer --e4m3 0x7D
./floatexplorer --e4m3 0x7E
./floatexplorer --e4m3 0x7F
./floatexplorer --e4m3 0x80
./floatexplorer --e4m3 0x81
./floatexplorer --e4m3 0x82
./floatexplorer --e4m3 0x83
./floatexplorer --e4m3 0x84
./floatexplorer --e4m3 0x85
./floatexplorer --e4m3 0x86
./floatexplorer --e4m3 0x87
./floatexplorer --e4m3 0x88
./floatexplorer --e4m3 0x89
./floatexplorer --e4m3 0x8A
./floatexplorer --e4m3 0x8B
./floatexplorer --e4m3 0x8C
./floatexplorer --e4m3 0x8D
./floatexplorer --e4m3 0x8E
./floatexplorer --e4m3 0x8F
./floatexplorer --e4m3 0x90
./floatexplorer --e4m3 0x91
./floatexplorer --e4m3 0x92
./floatexplorer --e4m3 0x93
./floatexplorer --e4m3 0x94
./floatexplorer --e4m3 0x95
./floatexplorer --e4m3 0x96
./floatexplorer --e4m3 0x97
./floatexplorer --e4m3 0x98
./floatexplorer --e4m3 0x99
./floatexplorer --e4m3 0x9A
./floatexplorer --e4m3 0x9B
./floatexplorer --e4m3 0x9C
./floatexplorer --e4m3 0x9D
./floatexplorer --e4m3 0x9E
./floatexplorer --e4m3 0x9F
./floatexplorer --e4m3 0xA0
./floatexplorer --e4m3 0xA1
./floatexplorer --e4m3 0xA2
./floatexplorer --e4m3 0xA3
./floatexplorer --e4m3 0xA4
./floatexplorer --e4m3 0xA5
./floatexplorer --e4m3 0xA6
./floatexplorer --e4m3 0xA7
./floatexplorer --e4m3 0xA8
./floatexplorer --e4m3 0xA9
./floatexplorer --e4m3 0xAA
./floatexplorer --e4m3 0xAB
./floatexplorer --e4m3 0xAC
./floatexplorer --e4m3 0xAD
./floatexplorer --e4m3 0xAE
./floatexplorer --e4m3 0xAF
./floatexplorer --e4m3 0xB0
./floatexplorer --e4m3 0xB1
./floatexplorer --e4m3 0xB2
./floatexplorer --e4m3 0xB3
./floatexplorer --e4m3 0xB4
./floatexplorer --e4m3 0xB5
./floatexplorer --e4m3 0xB6
./floatexplorer --e4m3 0xB7
./floatexplorer --e4m3 0xB8
./floatexplorer --e4m3 0xB9
./floatexplorer --e4m3 0xBA
./floatexplorer --e4m3 0xBB
./floatexplorer --e4m3 0xBC
./floatexplorer --e4m3 0xBD
./floatexplorer --e4m3 0xBE
./floatexplorer --e4m3 0xBF
./floatexplorer --e4m3 0xC0
./floatexplorer --e4m3 0xC1
./floatexplorer --e4m3 0xC2
./floatexplorer --e4m3 0xC3
./floatexplorer --e4m3 0xC4
./floatexplorer --e4m3 0xC5
./floatexplorer --e4m3 0xC6
./floatexplorer --e4m3 0xC7
./floatexplorer --e4m3 0xC8
./floatexplorer --e4m3 0xC9
./floatexplorer --e4m3 0xCA
./floatexplorer --e4m3 0xCB
./floatexplorer --e4m3 0xCC
./floatexplorer --e4m3 0xCD
./floatexplorer --e4m3 0xCE
./floatexplorer --e4m3 0xCF
./floatexplorer --e4m3 0xD0
./floatexplorer --e4m3 0xD1
./floatexplorer --e4m3 0xD2
./floatexplorer --e4m3 0xD3
./floatexplorer --e4m3 0xD4
./floatexplorer --e4m3 0xD5
./floatexplorer --e4m3 0xD6
./floatexplorer --e4m3 0xD7
./floatexplorer --e4m3 0xD8
./floatexplorer --e4m3 0xD9
./floatexplorer --e4m3 0xDA
./floatexplorer --e4m3 0xDB
./floatexplorer --e4m3 0xDC
./floatexplorer --e4m3 0xDD
./floatexplorer --e4m3 0xDE
./floatexplorer --e4m3 0xDF
./floatexplorer --e4m3 0xE0
./floatexplorer --e4m3 0xE1
./floatexplorer --e4m3 0xE2
./floatexplorer --e4m3 0xE3
./floatexplorer --e4m3 0xE4
./floatexplorer --e4m3 0xE5
./floatexplorer --e4m3 0xE6
./floatexplorer --e4m3 0xE7
./floatexplorer --e4m3 0xE8
./floatexplorer --e4m3 0xE9
./floatexplorer --e4m3 0xEA
./floatexplorer --e4m3 0xEB
./floatexplorer --e4m3 0xEC
./floatexplorer --e4m3 0xED
./floatexplorer --e4m3 0xEE
./floatexplorer --e4m3 0xEF
./floatexplorer --e4m3 0xF0
./floatexplorer --e4m3 0xF1
./floatexplorer --e4m3 0xF2
./floatexplorer --e4m3 0xF3
./floatexplorer --e4m3 0xF4
./floatexplorer --e4m3 0xF5
./floatexplorer --e4m3 0xF6
./floatexplorer --e4m3 0xF7
#./floatexplorer --e4m3 0xF8
#./floatexplorer --e4m3 0xF9
#./floatexplorer --e4m3 0xFA
#./floatexplorer --e4m3 0xFB
#./floatexplorer --e4m3 0xFC
#./floatexplorer --e4m3 0xFD
#./floatexplorer --e4m3 0xFE
#./floatexplorer --e4m3 0xFF
