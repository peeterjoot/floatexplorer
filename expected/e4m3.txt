value:    3
hex:      44
bits:     01000100
sign:     0
exponent:  1000                        (7 +1)
mantissa:      100
number:      1.100 x 2^(1)

